src/components/PaginationItem/PaginationItem.tsx

```diff
@@ -28,6 +28,7 @@ const PaginationItem = ({
         "is-active": isActive,
       })}
       onClick={onClick}
+      aria-current={isActive ? "page" : "false"}
```

Comment:
Have you double checked that `aria-current` is a valid attribute on buttons? I think I've only ever seen it being used on links.

Reply:
I've had a look around, I can't anywhere which specifies it can/can't be on a button. But I did fin this [article](https://tink.uk/using-the-aria-current-attribute/) which uses it on a `<td>`, so figured a button is also valid?

Reply:
OK awesome, thanks for checking this. Looks good 👍

Reply:
(Nitpicking) What you did is perfectly valid (`aria-current="false"` is not exposed to the user), but what do you think about simplifying this and returning undefined?

This will skip the `aria-current` attribute for items other than for the current page, like in the [example on MDN](https://developer.mozilla.org/en-US/docs/Web/Accessibility/ARIA/Attributes/aria-current#example).

```suggestion
      aria-current={isActive ? "page" : undefined}
```

Reply:
Done! Thanks for the review @petermakowski :)